client/my-sites/plans/jetpack-plans/plans-header.tsx

```diff
@@ -22,8 +23,14 @@ const StandardPlansHeader = () => (
 	<>
 		<FormattedHeader headerText={ translate( 'Plans' ) } align="left" brandFont />
 		<PlansNavigation path={ '/plans' } />
+		<h2 className="jetpack-plans__pricing-header">
+			{ preventWidows(
+				translate( 'Security, performance, and marketing tools made for WordPress' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Security, performance, and marketing tools made for WordPress' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/header-to-jetpack-plans-pages)